src/main/java/com/qunite/api/web/ResponseEntityControllerAdvice.java

```diff
@@ -49,7 +52,10 @@ public ResponseEntity<Map<String, String>> handleValidationError(
   @ExceptionHandler({
       ForbiddenAccessException.class,
       JWTDecodeException.class,
-      InvalidPasswordException.class})
+      InvalidPasswordException.class,
+      TokenExpiredException.class,
+      InvalidRefreshTokenException.class,
+      JWTVerificationException.class})
```

Comment:
As far as I am concerned, JWTVerificationException, TokenExpiredException and JWTDecodeException could be removed from this list because they are caught in JwtAuthorizationFilter try-catch

Reply:
Anyway you can remove JWTDecodeException and TokenExpiredException because they are sub-classes of JWTVerificationException